### PR TITLE
Fargate metadata

### PR DESCRIPTION
**Pull request by juan-coralogix**
Fargate integration now includes a RB script that pulls docker metadata from V4 Api  and appends it to each log.

### CODE REVIEW

File: fluentd/aws-ecs/image/fargate.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      record['docker'] = {
+        'id' => metadata['DockerId'],
+        'name' => metadata['Name'],
+        'container_hostname' => metadata['DockerName'],
+        'image' => metadata['Image'],
+        'image_id' => metadata['ImageID'],
+        'labels' => metadata['Labels']
+      }
+      record.delete('container_name')
+      record.delete('container_id')
```
Comment (giltk) [inline fluentd/aws-ecs/image/fargate.rb:42]: I think we should remove the two other fields that gets generated in the docker metadata:
- ecs_task_arn
- ecs_task_definition

Comment (giltk) [reply fluentd/aws-ecs/image/fargate.rb:42]: also i think docker.name and docker.container_hostname in fargate will always be the same name so we don't need both.

File: fluentd/aws-ecs/image/firelens.conf [hunk 2]
```diff
@@ -49,7 +56,7 @@
       @type memory
       compress gzip
       flush_thread_count 4
-      chunk_limit_size 6MB
+      chunk_limit_size 16MB
```
Comment (giltk) [inline fluentd/aws-ecs/image/firelens.conf:59]: why the limit size change?

**Review by giltk — CHANGES_REQUESTED** [review]

File: fluentd/aws-ecs/image/fargate.rb (hunk 1 above)
Comment (juan-coralogix) [reply fluentd/aws-ecs/image/fargate.rb:42]: task_definition I think is helpful due to the version. But what ever you think.

Comment (giltk) [reply fluentd/aws-ecs/image/fargate.rb:42]: we also have the version in another docker label so no need for the ecs metadata one.

Comment (juan-coralogix) [reply fluentd/aws-ecs/image/fargate.rb:44]: Removed the duplications.

**Comment by giltk** [comment]
@juan-coralogix creates a new branch to have less clutter

**Pull request closed without merging**